### PR TITLE
Message status

### PR DESCRIPTION
**Pull request by NaumVitanov**
Message status logic implementation

### CODE REVIEW

File: Database/Entities/MessageStatusEntity.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+		Status = status;
+		TimeStamp = timeStamp;
+	}
+
+	public int MessageId { get; set; }
+	public int ReceiverId { get; set; }
+	public int Status { get; set; }
+	public DateTime TimeStamp { get; set; }
+
+	[ForeignKey(nameof(MessageId))]
```
Comment (vgechev) [inline Database/Entities/MessageStatusEntity.cs:20]: Why are you using both FluentApi and annotations? =)

File: Services/Hubs/Enums/MessageStatus.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+﻿namespace Services.Hubs.Enums
+{
+	public enum MessageStatus
+	{
+		Send,
```
Comment (vgechev) [inline Services/Hubs/Enums/MessageStatus.cs]: Sent*, no?

File: Services/Hubs/Models/MessageStatusUpdateEvent.cs [hunk 3]
```diff
@@ -0,0 +1,5 @@
+﻿using Services.Hubs.Enums;
+
+namespace Services.Hubs.Models;
+
+public record MessageStatusUpdateEvent(int ChatId, int ReceiverId, MessageStatus Status, DateTime TimeStamp);
```
Comment (vgechev) [inline Services/Hubs/Models/MessageStatusUpdateEvent.cs]: Timestamp*

**Review by vgechev — APPROVED** [review]
LGTM, minor typos

File: Database/Entities/MessageStatusEntity.cs (hunk 1 above)
Comment (NaumVitanov) [reply Database/Entities/MessageStatusEntity.cs:20]: There is no way to create a composite PK with data annotations.

**Pull request merged**